app/graphql/enums/filter_match_enum.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Enums::FilterMatchEnum < Types::BaseEnum
+  value 'ABSENT', 'absent (negated present)', value: 'absent'
+  value 'CONTAINS', 'contains', value: 'contains'
+  value 'MATCHES', 'matches', value: 'matches'
```

Comment:
Can we stick with Hyacinth 2's "EQUALS" instead of changing it to "MATCHES"?  "MATCHES" makes me think we're supporting regular expressions.